app/models/wishlist.rb

```diff
@@ -0,0 +1,4 @@
+class Wishlist < ApplicationRecord
+  validates :name, length: { in: 3..30 }
+  validates :description, length: { in: 10..150 }
```

Comment:
Don't think it's necessary. It's up to the user, what he wants in his description.